docs/angular/guides/setup-mfe-with-angular.md

```diff
@@ -0,0 +1,393 @@
+
+In this article, I'll show you to how setup a Micro Frontend Architecture with Nx and Angular.
+
+## What we'll build
+
+We will put together a simple Admin Dashboard application that requires a user to log in to view the protected content.  
+To achieve this we will have two apps:
+
+- Admin Dashboard app
+  - This will contain the layout of the dashboard and the content that can only be viewed by an authenticated and authorized user.
+- Login app
+  - This will contain the forms and logic for authenticating and authorizing a user
+
+When the user tries to view a protected page within the Admin Dashboard, if they are not authenticated we will present them with a login form so that they can authenticate and view the contents of the page.
+
+The end result should look something like this:
+
+## First steps
+
+### Create an Nx Workspace
+
+To start with, we need to create a new Nx Workspace. We can do this easily with:  
+**npm**  
+`npx create-nx-workspace ng-mfe`
+
+**Yarn**  
+`yarn create nx-workspace ng-mfe --packageManager=yarn`
+
+You'll be prompted for a preset. We recommend selecting `empty` as it will allow you finer control over your workspace configuration.
+
+You'll also be prompted if you would like to setup Nx Cloud. For this tutorial select `No`, however, I highly recommend that you read more about it [here](https://nx.app/).
+
+### Add the Angular Plugin
+
+To add Angular-related features to our newly created monorepo we need to install the Angular Plugin. Again, this is pretty easy to do:
+
+_**NOTE:** Check that you are now at the root of your monorepo in your terminal. If not, run `cd ng-mfe`_
+
+**npm**  
+`npm install --save-dev @nrwl/angular`
+
+**Yarn**  
+`yarn add -D @nrwl/angular`
+
+Simple! You are now able to use Nx Generators to scaffold Angular applications and libraries.
+
+### Creating our apps
+
+Let's take a moment to think back to the architecture that we're trying to create.  
+We want to build a standalone Login application that can be independently deployed and consumed by other applications.  
+We also want to build an Admin Dashboard application that contains protected content that only an authorized user can view. This application should make use of the Login application to authorize the user.
+
+Therefore, we need to generate two applications. We also need to tell Nx that we want these applications to support Module Federation.
+
+We'll start with the Admin Dashboard application which will act as a host application for the MFE:
+
+**npm**  
+`npm run nx g @nrwl/angular:app dashboard --mfe --mfeType=host --routing=true`
+
+**Yarn**  
+`yarn nx g @nrwl/angular:app dashboard --mfe --mfeType=host --routing=true`
+
+You'll be prompted for some additional options. For this tutorial, just select the default options.
+The application generator will create and modify the files needed to setup an Angular application.
+
+Now, let's generate the Login application as a remote application.
```

Comment:
Maybe a little too complicated for a guide in my opinon :thinking:

Reply:
I think it's fine. 

I'd argue it's not too complicated, as it does some very basic operations in terms of Nx and Angular.

The logic never goes too deep either and it shows a great, potentially common, use case and as well as showing how to share state between the host and remote apps by forcing a shared library to be a singleton.

Reply:
Also, the blog post that Brandon is posting has my demo video which is even more basic than this.